### PR TITLE
Pin to ohai 8.10.0 for upcoming chef client release

### PR DESCRIPTION
**Pull request by chefsalim**
@chef/client-core @tyler-ball

### CODE REVIEW

**Comment by lamont-granquist** [comment]
pins should go into a 12.7-stable release branch, master should float on master.

**Comment by lamont-granquist** [comment]
sigh, nevermind...

:+1: 

forgot the whole release pipeline is constructed around the idea we only ship master, but we don't follow that so we have to commit pins to master every time we release...

File: omnibus/config/projects/chef.rb [hunk 1]
```diff
@@ -57,7 +57,7 @@
 
 # Chef Release version pinning
 override :chef, version: "local_source"
-override :ohai, version: "master"
+override :ohai, version: "8.10.0"
```
Comment (tyler-ball) [inline omnibus/config/projects/chef.rb]: This should be `v8.10.0` because that is what the [tag](https://github.com/chef/ohai/tree/v8.10.0) is named

**Comment by tyler-ball** [comment]
:+1: after my comment is addressed

**Comment by jkeiser** [comment]
What problem does this PR solve? I'm not intrinsically opposed, but not seeing what it actually does for us besides complicating our process.

**Pull request closed without merging**